doc/authoring_command_modules/README.md

```diff
@@ -134,9 +134,19 @@ $ az example --my-required-arg abc
 Testing
 -------
 
+Run all tests in a module:
+
+```
+python -m unittest discover -s <path_to_your_command_module>/tests
+```
+
+Run an individual test:
+
 ```
-python -m unittest discover -s <path_to_your_command_module>/azure/cli/command_modules/<module_name>/tests
+python <path_to_your_command_module>/<file> <class name>
```

Comment:
Note that this won't work unless the following snippet is in the test file:

```Python
if __name__ == '__main__':
    unittest.main()
```

Otherwise it will just drop you back to the command line.